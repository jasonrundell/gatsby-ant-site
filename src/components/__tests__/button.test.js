import React from 'react'
import renderer from 'react-test-renderer'

import Button from '../button'

describe('Button', () => {
  it('default renders correctly', () => {
    const tree = renderer.create(<Button>Default button</Button>).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('brand renders correctly', () => {
    const tree = renderer
      .create(<Button isBrand={true}>Default button</Button>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
