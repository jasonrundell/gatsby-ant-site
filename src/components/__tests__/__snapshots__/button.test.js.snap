// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button brand renders correctly 1`] = `
<button
  className="root brand"
>
  Default button
</button>
`;

exports[`Button default renders correctly 1`] = `
<button
  className="root"
>
  Default button
</button>
`;
